### ISSUE TITLE
Can I recombine the training set and test set when training DAMSM?

### ISSUE THREAD

**Issue by Curlyfuu** [taoxugit/AttnGAN#90]
I want to re-divide the dataset in the DAMSM stage, and use the originally divided dataset in the image generation stage. I don't know whether the results obtained in this way are convincing.
